compiler/ccgexprs.nim

```diff
@@ -1561,10 +1578,13 @@ proc genNewFinalize(p: BProc, e: PNode) =
   initLocExpr(p, e[1], a)
   initLocExpr(p, e[2], f)
   initLoc(b, locExpr, a.lode, OnHeap)
+
+  # xxx use `genTypeInfo`
```

Comment:
Should be addressed or removed in this PR

Reply:
fixed